stylesheets/utilities.scss

```diff
@@ -18,6 +18,14 @@
   outline: none;
 }
 
+.align-center {
```

Comment:
Primer calls it `.flex-items-center` https://primer.style/css/utilities/flexbox#align-items

and `.flex-justify-center`

https://primer.style/css/utilities/flexbox

Reply:
Thanks for review, I updated my PR changes.